apps/i18n/tutorialExplorer/et_ee.json

```diff
@@ -1,5 +1,6 @@
 {
-  "filterOrgNames": "Lisas kasutaja",
+  "startButton": "Alusta",
+  "filterOrgNames": "Autor",
```

Comment:
Are we okay with taking changes to other strings? (filterOrgNames)

Reply:
The strings in `i18n/locales` aren't used on the site at all. This is copying over strings that are already in `apps/i18n/tutorialExplorer`, so the only strings that will actually change with this is the `startButton`.